cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -0,0 +1,128 @@
+//  Copyright 2020  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+)
+
+const (
+	inflateFilePath  = "import_disk.wf.json"
+	inflateImagePath = "inflate_image.wf.json"
```

Comment:
Workflow file names are inconsistent with these constants. If we are to call import -> inflate (just the byte transfer), we should use it consistently.

Reply:
changed to inflate_disk.wf.json